string marker_name
uint8 pos_id